lua/nvim-tree/explorer/init.lua

```diff
@@ -151,11 +154,11 @@ function Explorer:reload(node, git_status)
 
       if not nodes_by_path[abs] then
         local new_child = nil
-        if t == "directory" and vim.loop.fs_access(abs, "R") and Watcher.is_fs_event_capable(abs) then
+        if type == "directory" and vim.loop.fs_access(abs, "R") and Watcher.is_fs_event_capable(abs) then
```

Comment:
To avoid any future refactor/rebase fails, and to keep git history simple, please rename `type` to `t`